pkgs/applications/networking/mailreaders/alot/default.nix

```diff
@@ -0,0 +1,62 @@
+{ stdenv, lib
+, pythonPackages, fetchFromGitHub, notmuch, file, gpgme
+, withManpage ? true }:
```

Comment:
This can be removed. It hardly takes any time to produce the manpages so we should.